lambdaInArrayList/LambdaInArraylistProgram.java

```diff
@@ -0,0 +1,29 @@
+package lambdaInArrayList;
+
+import java.util.*;
+
+public class LambdaInArraylistProgram {
+    //Java ArrayList Program to remove elements of Arraylist using lambda function.
+    public static void main(String[] args) {
+        ArrayList<String> arrayListObject = new ArrayList<String>();
+        arrayListObject.add("Anil");
```

Comment:
Optimize the add element to array

Reply:
Code is optimized for add element to array.